packages/shared/src/components/profile/ExperienceLevelDropdown.tsx

```diff
@@ -79,7 +79,7 @@ const ExperienceLevelDropdown = ({
         }}
         onOpenChange={setOpen}
         placeholder="Experience level"
-        icon={<TerminalIcon className="mr-2" />}
+        icon={<TerminalIcon className="ml-0 mr-2 !w-5" />}
```

Comment:
We have icon sizes, wouldn't the w-5 be IconSize.XSmall?

Reply:
Yes, but we also have `!w-6` class forced by the button so we need to override it :(

Reply:
We can use this but the icon will be slightly bigger the the previous 👀 

```tsx
icon={<TerminalIcon className="ml-0 mr-1" size={IconSize.XSmall} />}
```

<img width="728" alt="image" src="https://github.com/user-attachments/assets/43ef1f0b-c4b6-4140-b0db-5c377e5bd03b">

Reply:
But I think since the issue was alignment, then only `icon={<TerminalIcon className="ml-0 mr-1" />}` would work just fine, no?

Reply:
<img width="440" alt="Screenshot 2024-07-25 at 5 38 05 PM" src="https://github.com/user-attachments/assets/ca7b7fc3-aadd-4cdc-8049-755c4750558b">

Reply:
Okay, move to this approach 👍